os/kernel/Kconfig

```diff
@@ -1080,24 +1080,33 @@ config SCHED_LPWORKSTACKSIZE
 endif # SCHED_LPWORK
 endmenu # Work Queue Support
 
-menu "fault Manager"
-config FAULT_MGR
-	bool "Enable Fault Manager"
+menu "Binary Manager"
+config BINARY_MANAGER
+	bool "Enable Binary Manager"
 	default n
 	depends on APP_BINARY_SEPARATION
 	---help---
-		This interface is used to manage any fault occurs in a system.
-		It will either restart app or reboot the system according to
+		This interface is used to manage binaries.
+		It load/unload binaries and recover any fault occurs in a system.
+		It will either restart app or reboot the system according to 
 		the configuration.
 
-if FAULT_MGR
-config DEBUG_FAULT_MGR
-	bool "Enable Debug logs of Fault Manager"
+if BINARY_MANAGER
+config DEBUG_BINMGR_ERROR
```

Comment:
Most of the debug configs are in os/Kconfig.debug.
Is there any special reason for placing here?